{"uuid": "923d9ee8-d1b0-442e-96e1-50bba15b1f41", "children": ["88105894-990c-4e4a-84a7-b0b5bfec1b67", "08e3e265-1f79-48b5-9dd2-da8e0ef04ad4"], "befores": [{"name": "login", "status": "skipped", "statusDetails": {"message": "Skipped: ❌ Ошибка логина: Message: Поле ввода телефона недоступно\n\n", "trace": "  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/conftest.py\", line 434, in login\n    pytest.skip(f\"❌ Ошибка логина: {e}\")\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/a00059362/Desktop/Новая папка/onlineduken/.venv/lib/python3.13/site-packages/_pytest/outcomes.py\", line 159, in skip\n    raise Skipped(msg=reason, allow_module_level=allow_module_level)\n"}, "start": 1760003052922, "stop": 1760003068219}], "start": 1760003052922, "stop": 1760003073299}